hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/config/ThroughputProperties.java

```diff
@@ -0,0 +1,90 @@
+package pl.allegro.tech.hermes.frontend.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import pl.allegro.tech.hermes.frontend.publishing.handlers.ThroughputParameters;
+
+@ConfigurationProperties(prefix = "frontend.throughput")
+public class ThroughputProperties {
+
+    private String type = "unlimited";
+
+    private long fixedMax = Long.MAX_VALUE;
+
+    private long dynamicMax = Long.MAX_VALUE;
+
+    private long dynamicThreshold = Long.MAX_VALUE;
+
+    private long dynamicDesired = Long.MAX_VALUE;
+
+    private double dynamicIdle = 0.5;
+
+    private int dynamicCheckInterval = 30;
```

Comment:
it can be defined as Duration too, so it would be much more readable in the configuration properties, `30s` except `30` which is now quite vague